### DISCUSSION TITLE
Documentation is Updated ?

### DISCUSSION THREAD

**Discussion by AgnRakul** [GrapesJS/grapesjs#5113] [General]
Is there grapesjs Documentation is Updated ?

**Comment by artf** [comment]
To which part of documentation are you referring to?

**Reply by AgnRakul** [reply]
Some of the Methods are deprecated it shows while using it. i am not saying anything precisely. Current documentation is useful but if you updated with more examples of each and every methods and its use case it could be better right ?

**Reply by AgnRakul** [reply]
And if you update that most of them clear there doubts with documentation itself they wont create tickets on here its my opinion

**Reply by artf** [reply]
Sure but I don't know if adding examples/use cases for each method make sense but if you can indicate more precisely what could be improved that would help. Also curious what are the deprecated methods you've find

**Reply by RakulAgn** [reply]
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/76acbe81-1ba3-4c22-aa7c-816f12e096a7)
This is For a Example I Have to Check each and every methods and its use case

**Reply by artf** [reply]
There is a note and a link right to the deprecated tag
"Don't use named plugins, create plugins as simple functions. More about [Plugins](https://grapesjs.com/docs/modules/Plugins.html)"
